src/java/picard/sam/markduplicates/MarkDuplicates.java

```diff
@@ -64,28 +64,39 @@
-                    "The tool's main output is a new SAM or BAM file in which duplicates have been identified in the SAM flags field, or\n" +
-                    "optionally removed (see REMOVE_DUPLICATE and REMOVE_SEQUENCING_DUPLICATES), and optionally marked with a duplicate type\n" +
-                    "in the 'DT' optional attribute. In addition, it also outputs a metrics file containing the numbers of\n" +
-                    "READ_PAIRS_EXAMINED, UNMAPPED_READS, UNPAIRED_READS, UNPAIRED_READ DUPLICATES, READ_PAIR_DUPLICATES, and\n" +
-                    "READ_PAIR_OPTICAL_DUPLICATES.\n" +
-                    "\n" +
-                    "Usage example: java -jar picard.jar MarkDuplicates I=input.bam \\\n" +
-                    "                 O=marked_duplicates.bam M=marked_dup_metrics.txt\n" +
-                    "\n" +
-                    "The program can take either coordinate-sorted or query-sorted input, however the behavior is slightly different.\n" +
-                    "When the input is coordinate-sorted, unmapped mates of mapped records, and supplementary and secondary alignments are not\n" +
-                    "marked as duplicates. When the input is query-sorted (actually query-grouped) then unmapped mates get marked as their mapped\n" +
-                    "counter-parts, and secondary and supplementary reads get marked as the primary records with the same query-name.\n";
+    static final String USAGE_DETAILS = "<p>This tool locates and tags duplicate reads in a BAM or SAM file, where duplicate reads are " +
+            "defined as originating from a single fragment of DNA. Duplicates are identified as identical sequence alignments as indicated " +
+            "by their reference sequence dictionaries (in SAM header) and the 5' positions (POS in the alignment field of the SAM specification).  " +
+            "For additional information on the SAM specification, please see this <a href='https://samtools.github.io/hts-specs/SAMv1.pdf'>document</a>.</p>" +
+            "" +
+            "<p>Duplicates can arise during sample preparation e.g. library construction using PCR or as optical duplicates.  " +
+            "For information on the former, please see the documentation for Picard's <a href='https://broadinstitute.github.io/picard/command-line-overview.html#EstimateLibraryComplexity'>EstimateLibraryComplexity</a> tool.  " +
+            "The latter duplication artifcats result from reads derived from a single amplification cluster, incorrectly detected as multiple clusters by the optical sensor of the sequencing instrument.  </p>" +
+            "" +
+            "<p>The READ_NAME_REGEX and the OPTICAL_DUPLICATE_PIXEL_DISTANCE options are the primary methods to identify and differentiate duplicate types.  " +
+            "The tool's main output is a new SAM or BAM file, in which duplicates have been identified in the SAM flags field and can be marked as such using the optional 'DT' argument.  " +
```

Comment:
"duplicates have been identified in the SAM flags field and can be marked as such using the optional 'DT' argument" -> Confusing. Be a bit more specific about what is done to the SAM flag, and make the statement about DT separate if you want to mention that. Right now it sounds like by default we only *identify* (meaning find) duplicates, but don't "mark them as such" unless requested with DT, which is wrong.